packages/remix-template/app/routes/index.tsx

```diff
@@ -1,23 +1,31 @@
+import styles from "~/styles/index.css";
+
+export function links() {
+  return [{ rel: "stylesheet", href: styles }];
+}
+
 export default function Index() {
   return (
     <div style={{ fontFamily: "system-ui, sans-serif", lineHeight: "1.4" }}>
-      <h1>Welcome to Remix</h1>
+      <h1>Congratulations!</h1>
+      <p>
+        You’ve successfully created your <a target="_blank" href="https://remix.run/">Remix</a> project for <a target="_blank" href="https://developer.fastly.com/learning/compute/javascript/">Fastly Compute@Edge</a>.
+      </p>
+      <p>
+        If you already have a Remix application that you'd like to move to Compute@Edge, see
+        our <a target="_blank" href="https://github.com/fastly/remix-compute-js/blob/main/MIGRATING.md">Migration Guide</a>.
```

Comment:
Would suggest lowercase for _migration guide_.

Reply:
ok